File: Javascript (NODE JS)/planina.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.on('line', (line) => {
    var nums = line.split(' ');
    var a = parseInt(nums[0]);
    b = Math.pow(2,a)   
    b += 1
    c = b*b
    console.log(c)
    
});

File: C#/Planina.cs
using System.IO;
using System;

class Program
{
    static void Main()
    {
        string line;
        line = Console.ReadLine();
        long a = Int64.Parse(line);
        double b = Math.Pow(2,a);
        b ++; 
        Console.WriteLine(b*b);
        
        
    }
}
